### DISCUSSION TITLE
Hide the Map layout for schemas that do not have geographic fields

### DISCUSSION THREAD

**Discussion by daawaan4x** [directus/directus#13039] [Feature Requests]
Currently, maps are visible for schemas that do not have geographic fields, even in the files module and the users module. There should be a way to introspect the schema first to check if it has geographic fields before showing the map layout as an option

**Comment by benhaynes** [comment]
I agree. Hiding, or at least disabling would be nice if there are no fields in the schema to support the main integration.

For Files/Users, you can add location fields, so all this does indeed need to be live with introspection... and based on the fields you have access to in permissions.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
